coopersystem-fsd/pcc
# Add course private tabs

## Description

- Added private tab when event type is course or past course and has a child event;
- Added online event formats;
- Fixed navbar button that turned white when the current page was the same as the dropdown menu.

## Steps to test

1. Create a Course type event;
2. Fill in the class fields

**Expected behavior:** The fields will be displayed on the event page in the sidebar.

## Additional information
